RELEASING.md

```diff
@@ -11,9 +11,9 @@ boms/cloud-oss-bom/RELEASING.md) is not part of this document.
 
 | ArtifactId | Rapid project name | Dependency |
```

Comment:
"Artifact ID" since this is not code
Rapid projcet name --> Rapid project

Reply:
Fixed.